docs/eks.md

```diff
@@ -28,9 +28,9 @@ can use `aws configure` to set them up.
 2. Define the cluster name, region and IAM role name in the environment variables:
 
     ```shell
-    export NAME="promscale-${USER}-$(date +%s)"
-    export REGION=us-east-1
-    export ROLE=${USER}-amazoneks_ebs_csi_driverrole
+    export NAME="${USER}-promscale-$(date +%F-%H-%M)"
+    export REGION="us-east-1"
+    export ROLE="${USER}-amazoneks_ebs_csi_driverrole-$(date +%F-%H-%M)"
```

Comment:
How about using the NAME in ROLE? Yeah, it would bit lengthy :)

```suggestion
    export ROLE="${NAME}-amazoneks_ebs_csi_driverrole"
```

Reply:
Good idea! I amended my commit and made the suffix a bit shorter but hopefully still informative enough.